src/main/java/minicraft/core/Game.java

```diff
@@ -59,6 +70,12 @@ public static void exitDisplay() {
 	static boolean running = true;
 	public static void quit() { running = false; }
 
+	public static final ReloadableResourceManager resourceManager = new ReloadableResourceManager();
+	public static ResourcePackManager resourcePackManager;
+	public static VanillaResourcePack defaultResourcePack;
+	public static boolean reloading;
```

Comment:
The handler could just handle the displays by their corresponding classes. For the static classes, I think it is a big change for the game, so I think it is not the time for this.

Reply:
Here, the display class for resource packs is `ResourcePackDisplay`, so it could handle the displays related to the resource pack system. About the display stack, if you mean the stacking of displays with handlers, it has already been implemented by an `ArrayDeque`.